sdk/tests/utilities/id_generator.py

```diff
@@ -0,0 +1,66 @@
+import uuid
+
+
+class IdGenerator:
+
+    default_scope = "sdk_example"
+
+    def __init__(self, scope="sdk_example"):
```

Comment:
should this be `scope=self.default_scope`? or `scope=default_scope`? I forget which is correct in this case.

Reply:
goot spot